CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v2.3.0](https://github.com/puppetlabs/puppet_operational_dashboards/tree/v2.3.0) (2024-02-03)
+## [v2.3.0](https://github.com/puppetlabs/puppet_operational_dashboards/tree/v2.3.0) (2024-02-05)
 
-[Full Changelog](https://github.com/puppetlabs/puppet_operational_dashboards/compare/v2.2.0...v2.3.0)
+[Full Changelog](https://github.com/puppetlabs/puppet_operational_dashboards/compare/temp_tag...v2.3.0)
+
+## [temp_tag](https://github.com/puppetlabs/puppet_operational_dashboards/tree/temp_tag) (2024-02-03)
```

Comment:
`temp_tag`?

Reply:
some gubbins in there from last year not sure why